### PR TITLE
Kazoo 4141 - parse modules for kapps_config usages

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

**Comment by jamesaimonetti** [comment]
@fenollp your validate-js script is borked. Please fix it to either not bork itself or provide meaningful output to the user to fix the issue. Ain't nobody got time for reading Python :)

**Comment by jamesaimonetti** [comment]
@fenollp you're in luck; I had time for dat... :) Let me know what you think of the change to validate-js but it seems to pass muster.

File: scripts/generate-schemas.escript [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env escript
+%%! -sname kazoo_xref
```
Comment (fenollp) [inline scripts/generate-schemas.escript]: Remove this line

File: scripts/generate-schemas.escript [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+-module(generate_schemas).
+-mode('compile').
+-compile([debug_info]).
+-compile({'no_auto_import', [get/0]}).
+
+-export([main/1]).
+
+main(_) ->
+    %% {'ok', CWD} = file:get_cwd(),
```
Comment (fenollp) [inline scripts/generate-schemas.escript]: Why is this code dead?

File: core/kazoo_ast/src/cb_api_endpoints.erl [hunk 3]
```diff
@@ -0,0 +1,731 @@
...
+
+path_name(Module) when is_atom(Module) ->
+    path_name(kz_util:to_binary(Module));
+path_name(<<_/binary>>=Module) ->
+    case re:run(Module
+               ,<<"^cb_([a-z_]+?)(?:_(v[0-9]))?$">>
+               ,[{'capture', 'all_but_first', 'binary'}]
+               )
+    of
+        {'match', [<<"about">>]} -> <<?CURRENT_VERSION/binary, "/about">>;
```
Comment (fenollp) [inline core/kazoo_ast/src/cb_api_endpoints.erl]: What is this doing here? It was removed earlier this week. Did you really `git mv` the file?

File: applications/crossbar/doc/ref/about.md [hunk 4]
```diff
@@ -8,11 +8,11 @@
...
 #### Fetch
 
-> GET /v2/accounts/{ACCOUNT_ID}/about
+> GET /v2/about
 
 ```curl
 curl -v -X GET \
     -H "X-Auth-Token: {AUTH_TOKEN}" \
-    http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/about
+    http://{SERVER}:8000/v2/about
```
Comment (fenollp) [inline applications/crossbar/doc/ref/about.md]: This is a lie. Without accountid it returns 403. Please revert the "about" change you made all sneakily.
Please make multiple small PRs for unrelated things.

**Comment by fenollp** [comment]
Merged to help CM. However there are a couple non blocking things that should be addressed.

File: core/kazoo_ast/src/cb_api_endpoints.erl (hunk 3 above)
Comment (fenollp) [inline core/kazoo_ast/src/cb_api_endpoints.erl]: Fixed in #2369

File: applications/crossbar/doc/ref/about.md (hunk 4 above)
Comment (fenollp) [inline applications/crossbar/doc/ref/about.md]: Fixed in #2369

File: scripts/generate-schemas.escript (hunk 1 above)
Comment (fenollp) [inline scripts/generate-schemas.escript]: Fixed in #2369

**Pull request merged**